iib/workers/tasks/build.py

```diff
@@ -535,9 +536,11 @@ def _prepare_request_for_build(
         with set_registry_token(overwrite_from_index_token, from_index):
             from_index_resolved = _get_resolved_image(from_index)
             from_index_arches = _get_image_arches(from_index_resolved)
+            ocp_version = get_image_label(from_index_resolved, 'com.redhat.index.delivery.version') or 'v4.5'
```

Comment:
Do you think the default label value should be configurable? Could this ever change in the future as v4.5 goes EoL?

Reply:
probably depends on if this is intended to be used by other folks, but once we set the label name, I don't think that it should change. 

Even as 4.5 goes EOL it would require us to convert 4.6 index image's to the new value. 

If we were to switch regardless of config value, you would need to write a migration step into the code I think. Is this required t be handled right now?

Reply:
It's not required. I was just checking with you to make sure.